consensus/debugging.go

```diff
@@ -36,6 +36,14 @@ func (m *consensusModule) SetUtilityContext(utilityContext modules.UtilityContex
 	m.utilityContext = utilityContext
 }
 
+func (m *consensusModule) SetAggregatedMetadata(maxHeight, minHeight uint64, peerAddress string) {
```

Comment:
Are we using this anywhere?

Reply:
I left this for the issue #352,  and for testing. I will remove it, and I will try to avoid adding it back, in parallel to your suggestion of not having more than necessary shared functions.